File: beginner/1008/salary.c
#include <stdio.h>

int	main(void)
{
	int		worker_id;
	int		worker_hours;
	double	worker_hour_rate;
	double	worker_salary;

	scanf("%d %d %lf", &worker_id, &worker_hours, &worker_hour_rate);
	worker_salary = worker_hours * worker_hour_rate;
	printf("NUMBER = %d\n", worker_id);
	printf("SALARY = U$ %.2lf\n", worker_salary);
}

File: beginner/1008/Salary.java
// challenge link - https://judge.beecrowd.com/en/problems/view/1008

import java.io.IOException;
import java.util.Scanner;

public class Salary {
	public static void main(String[] args) throws IOException {
		Scanner reader = new Scanner(System.in);
		int workerID = reader.nextInt();
		int hoursWorked = reader.nextInt();
		double hourlyRate = reader.nextDouble();
		System.out.println("NUMBER = " + workerID);
		System.out.println("SALARY = U$ " + String.format("%.2f", hoursWorked * hourlyRate));
		reader.close();
	}
}
